bootstrap.js

```diff
@@ -53,64 +57,64 @@ function include(src) {
 }
 
 function setDefaultPrefs() {
-  let branch = Services.prefs.getDefaultBranch("");
+  // let branch = Services.prefs.getDefaultBranch('');
```

Comment:
We've got the previous versions if we need the old code, so we should just delete it instead of commenting it out.  😃